packages/mermaid/src/utils/sanitizeDirective.ts

```diff
@@ -0,0 +1,84 @@
+import { configKeys } from '../defaultConfig.js';
+import { log } from '../logger.js';
+
+/**
+ * Sanitizes directive objects
+ *
+ * @param args - Directive's JSON
+ */
+export const sanitizeDirective = (args: any): void => {
```

Comment:
Because `utils.ts` had this nifty line at the top, it didn't matter what the type was.
https://github.com/mermaid-js/mermaid/blob/6e51f8fd9840a788bcca1f2cd2aabd78bcf3b74a/packages/mermaid/src/utils.ts#L1

Changing it to unknown adds ~10 errors, which are not really relevant in our case as the args is basically an object | array, hence `any`.

<img width="443" alt="image" src="https://github.com/mermaid-js/mermaid/assets/10703445/c76ffaa5-bf64-4181-8822-355f3b882b4a">

Reply:
> Because `utils.ts` had this nifty line at the top, it didn't matter what the type was
>
> https://github.com/mermaid-js/mermaid/blob/6e51f8fd9840a788bcca1f2cd2aabd78bcf3b74a/packages/mermaid/src/utils.ts#L1

:rofl: Oh well, `any` is fine then! `unknown` would be even better (especially for security-based code), but if we're not changing existing code, there's no need to change the typing.